taskcluster/ci/fetch/kind.yml

```diff
@@ -13,13 +13,13 @@ task-defaults:
     docker-image: {in-tree: base}
 
 tasks:
-    android-sdk-3859397:
+    android-sdk-9477386:
         description: Android SDK
         fetch:
             type: static-url
-            url: https://dl.google.com/android/repository/sdk-tools-linux-3859397.zip
```

Comment:
Thank you so much for updating this! I know we have a poor history of upgrading this part of the Android tooling. I'm glad it doesn't break anything because no one's touched it in 3 years 😅